Make Install fails with - ImportError: cannot import name sanitizer

This project does not deploy successfully anymore. This had worked on the 28/06/2016 but when I revisited today and run `make install` the deployment eventually fails with the below error.

```
env/bin/python manage.py migrate --noinput --no-initial-data
/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/reversion/models.py:6: RemovedInDjango19Warning: django.contrib.contenttypes.generic is deprecated and will be removed in Django 1.9. Its contents have been moved to the fields, forms, and admin submodules of django.contrib.contenttypes.
  from django.contrib.contenttypes import generic

/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/django/contrib/admin/util.py:7: RemovedInDjango19Warning: The django.contrib.admin.util module has been renamed. Use django.contrib.admin.utils instead.
  "Use django.contrib.admin.utils instead.", RemovedInDjango19Warning)

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 328, in execute
    django.setup()
  File "/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/django/apps/config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/djangocms_text_ckeditor/models.py", line 13, in <module>
    from .html import clean_html, extract_images
  File "/home/vagrant/django-cms-demo/env/local/lib/python2.7/site-packages/djangocms_text_ckeditor/html.py", line 8, in <module>
    from html5lib import sanitizer, serializer, treebuilders, treewalkers
ImportError: cannot import name sanitizer
Makefile:54: recipe for target 'migrate' failed
make[2]: *** [migrate] Error 1
make[2]: Leaving directory '/home/vagrant/django-cms-demo'
Makefile:30: recipe for target 'update' failed
make[1]: *** [update] Error 2
make[1]: Leaving directory '/home/vagrant/django-cms-demo'
Makefile:20: recipe for target 'install' failed
make: *** [install] Error 2
```

Looking at the commit history I see that the package reference for [https://github.com/divio/djangocms-text-ckeditor](https://github.com/divio/djangocms-text-ckeditor) changed to point to master. In the light of this I guess that the problem is probably introduced at the **djangocms-text-ckeditor** project.

--- a/requirements.txt
+++ b/requirements.txt
@@ -2,6 +2,8 @@ Django>=1.8.8,<1.9
 django-reversion>=1.8.2,<1.8.6
 # django-reversion>=1.10,<1.11
 psycopg2>=2.6.1  # required for django.db.backends.postgresql_psycopg2
+# https://github.com/divio/djangocms-text-ckeditor/pull/335
+html5lib>=0.90,!=0.9999,!=0.99999,<0.99999999
 
 # we can modify the django CMS installation at will to test certain features
 https://github.com/divio/django-cms/archive/develop.zip#egg=django-cms

